apps/emqx_cluster_link/test/emqx_cluster_link_config_SUITE.erl

```diff
@@ -12,6 +12,9 @@
 -compile(export_all).
 -compile(nowarn_export_all).
 
+-define(BASE_CLINK_MQTT_PORT, 11883).
```

Comment:
Nit, since `mk_cluster` seems to be always called with `N >= 1` and we always add `N * 10_000` to this:

```suggestion
-define(BASE_CLINK_MQTT_PORT, 1883).
```